internal/cli/cmd/bench/pgbench.go

```diff
@@ -44,56 +42,48 @@ const (
 )
 
 var pgbenchExample = templates.Examples(`
-# pgbench run on a cluster, that will exec all steps, cleanup, prepare and run
-kbcli bench pgbench mytest --cluster pgcluster --database postgres --user xxx --password xxx
-
-# pgbench run on a cluster with cleanup, just exec cleanup that will delete the testdata
-kbcli bench pgbench cleanup mytest --cluster pgcluster --database postgres --user xxx --password xxx
-
-# pgbench run on a cluster with prepare, just exec prepare that will create the testdata
-kbcli bench pgbench prepare mytest --cluster pgcluster --database postgres --user xxx --password xxx
-
-# pgbench run on a cluster with run, just exec run that will run the test
-kbcli bench pgbench run mytest --cluster pgcluster --database postgres --user xxx --password xxx
-
-# pgbench run on a cluster with  threads and  client count
-kbcli bench sysbench mytest --cluster pgcluster --user xxx --password xxx --database xxx --clients 5 --threads 5
-
-# pgbench run on a cluster with specified transactions
-kbcli bench pgbench mytest --cluster pgcluster --database postgres --user xxx --password xxx --transactions 1000
-
-# pgbench run on a cluster with specified seconds
-kbcli bench pgbench mytest --cluster pgcluster --database postgres --user xxx --password xxx --duration 60
-
-# pgbench run on a cluster with select only
-kbcli bench pgbench mytest --cluster pgcluster --database postgres --user xxx --password xxx --select
+	# pgbench run on a cluster, that will exec all steps, cleanup, prepare and run
+	kbcli bench pgbench mytest --cluster pgcluster --database postgres --user xxx --password xxx
+	
+	# pgbench run on a cluster with cleanup, just exec cleanup that will delete the testdata
```

Comment:
just exec cleanup that will delete the testdata -> only cleanup by deleting the testdata